.github/workflows/test.yml

```diff
@@ -0,0 +1,33 @@
+name: Tests
+on: [push, pull_request]
+
+jobs:
+  run:
+    name: Node ${{ matrix.node }}
+    runs-on: ubuntu-latest
+
+    strategy:
+      fail-fast: false
+      matrix:
+        node: [8, 10, 12]
+
+    steps:
+      - name: Clone repository
+        uses: actions/checkout@v1
+
+      - name: Set Node.js version
+        uses: actions/setup-node@v1
+        with:
+          node-version: ${{ matrix.node }}
+
+      - run: node --version
+      - run: npm --version
+      - run: git --version
+
+      - name: Install npm dependencies
+        run: npm ci
```

Comment:
`npm install` because there is no package-lock

Reply:
Yeah, this isn't final. I plan to merge #61 first, I just want to understand what is needed for tests to work first